Add features section (#22)
inveniosoftware/docs-invenio-ils: docs/features/circulation.md

> # Loan Circulation with Delivery Options and State Tracking

Reviewer: ```suggestion
# Loan Circulation with delivery options and state tracking
```

Reviewer 2: Agree, I noticed the camel case in the titles in the other PRs too. To be lowercased everywhere

> # Loan Circulation with Delivery Options and State Tracking

Reviewer 2: ```suggestion
# Circulation
```

I would explain a little bit more here, basically briefly listing all the circulation module features.
It feels like you are immediately jumping to the loan extension, while the core of the module is really the checkout/checking and the various states.

> Invenio ILS provides comprehensive tools for managing loan circulation, including tracking and flexible delivery options.

Reviewer 2: Everywhere
```suggestion
InvenioILS provides comprehensive tools for managing loan circulation, including tracking and flexible delivery options.
```

## Loan Extensions

> Users have the option to request extensions on their loans through the system. This ensures they can keep borrowed items for as long as they need them within the library's policies, which are customizable to fit the library's specific needs.

Reviewer: ```suggestion
Users have the option to request extensions on their loans. This ensures they can keep borrowed items for as long as they need them within the library's policies, which are customizable to fit the library's specific needs.
```
